tests/integration/sdk_api/alerts/test_alerts.py

```diff
@@ -283,6 +283,28 @@ def test_alert_templates(self):
             alerts[0], project_name=project_name, alert_name=alert_name
         )
 
+    @pytest.mark.parametrize(
+        "alert_name_in_config, alert_name_as_func_param",
+        [
+            (None, None),
+            (None, ""),
+            ("", None),
+            ("", ""),
+        ],
+    )
+    def test_store_alert_config_missing_alert_name(
+        self, alert_name_in_config, alert_name_as_func_param
+    ):
+        project_name = "my-project"
+        project = mlrun.new_project(project_name)
+        alert_data = mlrun.alerts.alert.AlertConfig(
+            name=alert_name_in_config, project=None
+        )
+        with pytest.raises(
+            mlrun.errors.MLRunInvalidArgumentError, match="Alert name must be provided"
+        ):
+            project.store_alert_config(alert_data, alert_name=alert_name_as_func_param)
+
```

Comment:
Integration tests tend to have a long runtime, so we typically avoid using parametrize to prevent further extending the execution time. However, I believe this test would be better suited as a unit test under `httpdb` tests, rather than remaining in the projects tests as integration.

Reply:
@yaelgen Which exact file are you referring to? Most unit tests for httpdb that I’ve seen only interact with sqldb.py or nopdb.py, not the httpdb file itself.